### PR TITLE
Update Installation Guide

### PR DESCRIPTION
**Pull request by YvonneW**
Following files pertaining to **Installation Guide** were renamed and updated:  
- `installation-guide.md`
- `installing-the-product.md`
- `installation-prerequisites.md`
- `installing-on-linux-or-os-x.md`
- `installing-on-solaris.md`
- `installing-on-windows.md`
- `installing-as-a-linux-service.md`
- `installing-as-a-windows-service.md`
- `getting-started-with-the-management-console.md`
- `customizing-the-management-console.md`
- `shutting-down-and-restarting-the-server.md`

Following images pertaining to **Installation Guide** were renamed and moved to the appropriate directory:  
- `attachments/26838936/27042108.png` ==> `../../assets/img/setup/echo-command.png`
- `attachments/thumbnails/26838941/27042151` ==> `../../assets/img/setup/properties-`option.png
- `attachments/26838941/27042150.png` ==> `../../assets/img/setup/advanced-tab.png`
- `attachments/thumbnails/26838941/27042154` ==> `../../assets/img/setup/new-button.png`
- `attachments/103328963/103328964.png` ==> `../../assets/img/setup/edit-system-variable-window.png`
- `attachments/28717183/29364286.png` ==> `../../assets/img/setup/console-output.png`
- `attachments/28717183/29364285.png` ==> `../../assets/img/setup/service-installation-confirmation.png`
- `attachments/28717183/29364288.png` ==> `../../assets/img/setup/service-startup-message.png`
- `attachments/28717183/29364290.png` ==> `../../assets/img/setup/service-stop-message.png`
- `attachments/28717183/29364291.png` ==> `../../assets/img/setup/service-removal-message.png`
- `attachments/34611935/44179743.png` ==> `../../assets/img/setup/server-link.png`
- `attachments/103328985/103328992.png` ==> `../../assets/img/setup/management-console.png`
- `attachments/103328985/103328990.png` ==> `../../assets/img/setup/main-menu.png`
- `attachments/103328985/103328991.png` ==> `../../asstes/img/setup/entitlement-menu-item.png`
- `attachments/103328985/103328986.png` ==> `../../assets/img/setup/manage-menu-png`
- `attachments/103328985/103328987.png` ==> `../../assets/img/setup/monitor-menu.png`
- `attachments/103328985/103328989.png` ==> `../../assets/img/menu/tools-menu.png`
- `attachments/103328970/103328971.png` ==> `../../assets/img/menu/tools-menu.png` 
- `attachments/103328985/103328988.png` ==> `../../assets/img/setup/configure-menu.png`
- `../../assets/img/setup/shutdown-restart.png`

### CODE REVIEW

File: en/docs/setup/installing-on-linux-or-os-x.md [hunk 1]
```diff
@@ -109,15 +110,12 @@ you can take the following approaches:
...
-    .
-
-When using SUSE Linux, it ignores `          /etc/resolv.conf         `
-and only looks at the `          /etc/hosts         ` file. This means
-that the server will throw an exception on startup if you have not
-specified anything besides localhost. To avoid this error, add the
-following line above `          127.0.0.1 localhost         ` in the
-`          /etc/hosts         ` file: `          <ip_address>         `
-`          <machine_name> localhost         `
+    vault](../../admin-guide/carbon-secure-vault-implementation).
```
Comment (sherenewso2) [inline en/docs/setup/installing-on-linux-or-os-x.md]: this should change to ../../administer

File: en/docs/setup/installing-on-solaris.md [hunk 2]
```diff
@@ -1,12 +1,10 @@
 # Installing on Solaris
 
-!!! tip
-    
-    **Before you begin**, [please see our compatibility
-    matrix](https://docs.wso2.com/display/compatibility/Tested+Operating+Systems)
+!!! tip "Before you begin"
+    See the [compatibility matrix](../../admin-guide/tested-operating-systems)
```
Comment (sherenewso2) [inline en/docs/setup/installing-on-solaris.md]: this should change to "administer"

File: en/docs/setup/getting-started-with-the-management-console.md [hunk 3]
```diff
@@ -66,102 +63,94 @@ divided into different sections.
...
 
-**Related links**
+!!! info "Related links" 
+    See the following topics for instructions on how to use each of these
+    menu items.
 
-See the following topics for instructions on how to use each of these
-menu items.
+    -   See [Configuring Users, Roles and
+        Permissions](../../using-wso2-identity-server/configuring-users-roles-and-permissions) for
```
Comment (sherenewso2) [inline en/docs/setup/getting-started-with-the-management-console.md]: "using-wso2-identity-server" should change to "learn"

File: en/docs/setup/getting-started-with-the-management-console.md [hunk 4]
```diff
@@ -66,102 +63,94 @@ divided into different sections.
...
-    configuring identity providers.
+    -   See [Configuring the Realm](../../using-wso2-identity-server/configuring-the-realm) for configuring
+        user stores.
+    -   See [Claim Management](../../using-wso2-identity-server/claim-management) for configuring claims.
+    -   See [Adding and Configuring a Service
+        Provider](../../using-wso2-identity-server/adding-and-configuring-a-service-provider) for
+        configuring service providers.
+    -   See [Adding and Configuring an Identity
+        Provider](../../using-wso2-identity-server/adding-and-configuring-an-identity-provider) for
+        configuring identity providers.
```
Comment (sherenewso2) [inline en/docs/setup/getting-started-with-the-management-console.md]: change these link paths to "learn"

File: en/docs/setup/getting-started-with-the-management-console.md [hunk 5]
```diff
@@ -66,102 +63,94 @@ divided into different sections.
...
+        for information on configuring challenge questions. Also see [Using
+        the End User Dashboard](../../using-wso2-identity-server/using-the-end-user-dashboard) for
+        information on challenge questions.
+    -   See [Email Templates](../../using-wso2-identity-server/email-templates) for configuring email
+        templates. Also see [Customizing Automated
+        Emails](../../using-wso2-identity-server/customizing-automated-emails)
+        for information on how to customize these templates.
+    -   See [Creating New Keystores in the WSO2 Administration
+        Guide](../../admin-guide/creating-new-keystores)
+        for configuring keystores.
```
Comment (sherenewso2) [inline en/docs/setup/getting-started-with-the-management-console.md]: change the link paths according to the new folder structure

File: en/docs/setup/getting-started-with-the-management-console.md [hunk 6]
```diff
@@ -66,102 +63,94 @@ divided into different sections.
...
 
 The monitor menu includes a list of features focused on providing logs
 and statistics related to monitoring the Identity Server. For more
 information on these features and their usage, see the topics on
-[monitoring the Identity Server](Monitoring_the_Identity_Server).
 
-![](attachments/103328985/103328987.png)
+[monitoring the Identity Server](../../using-wso2-identity-server/monitoring-the-identity-server).
+
+![Monitor menu](../../assets/img/setup/monitor-menu.png)
```
Comment (sherenewso2) [inline en/docs/setup/getting-started-with-the-management-console.md]: change link path to "learn"

File: en/docs/setup/getting-started-with-the-management-console.md [hunk 7]
```diff
@@ -66,102 +63,94 @@ divided into different sections.
...
-    Management](Using_Workflows_with_User_Management) for information
-    on Workflow Engine Profiles.
--   See [Creating and Managing Tenants](Creating_and_Managing_Tenants)
-    for information on multitenancy.
+    -   See [Server Roles](../../using-wso2-identity-server/server-roles) for information on server roles.
+    -   See [Using Workflows with User
+        Management](../../using-wso2-identity-server/using-workflows-with-user-management) for information
+        on Workflow Engine Profiles.
+    -   See [Creating and Managing Tenants](../../using-wso2-identity-server/creating-and-managing-tenants)
+        for information on multitenancy.
```
Comment (sherenewso2) [inline en/docs/setup/getting-started-with-the-management-console.md]: change link paths to "learn"

File: en/docs/setup/getting-started-with-the-management-console.md [hunk 8]
```diff
@@ -66,102 +63,94 @@ divided into different sections.
...
 
 ### Tools menu
 
 The tools menu includes SAML and XACML tools. For more details on each
-of these tools and their usage, see the topics on [working with
-tools](Using_Tools).
+of these tools and their usage, see the topics on [working with tools](../../using-wso2-identity-server/using-tools).
```
Comment (sherenewso2) [inline en/docs/setup/getting-started-with-the-management-console.md]: change link paths to "learn"

File: en/docs/setup/getting-started-with-the-management-console.md (hunk 4 above)
Comment (YvonneW) [reply en/docs/setup/getting-started-with-the-management-console.md]: Fixed

File: en/docs/setup/getting-started-with-the-management-console.md (hunk 5 above)
Comment (YvonneW) [reply en/docs/setup/getting-started-with-the-management-console.md]: Fixed

File: en/docs/setup/getting-started-with-the-management-console.md (hunk 6 above)
Comment (YvonneW) [reply en/docs/setup/getting-started-with-the-management-console.md]: Fixed

File: en/docs/setup/getting-started-with-the-management-console.md (hunk 7 above)
Comment (YvonneW) [reply en/docs/setup/getting-started-with-the-management-console.md]: Fixed

File: en/docs/setup/installation-prerequisites.md [hunk 9]
```diff
@@ -41,28 +35,31 @@ development platforms before starting the installation.
...
-<li>All WSO2 Carbon-based products are generally compatible with most common DBMSs. The embedded H2 database is suitable for development and testing. For enterprise production environments we recommend an industry-standard RDBMS such as Oracle, PostgreSQL, MySQL, MS SQL, etc. For more information, see <a href="https://docs.wso2.com/display/ADMIN44x/Working+with+Databases">Working with Databases</a> .</li>
+<li>All WSO2 Carbon-based products are generally compatible with most common DBMSs. The embedded H2 database is suitable for development and testing. For enterprise production environments we recommend an industry-standard RDBMS such as Oracle, PostgreSQL, MySQL, MS SQL, etc. For more information, see <a href="../../admin-guide/working-with-databases">Working with Databases</a> .</li>
 <li>WSO2 Identity Server supports using any of the following as a user store :
 <ul>
 <li>RDBMS</li>
 <li>An LDAP such as OpenLDAP</li>
 <li>Active Directory</li>
-<li><a href="_Configuring_User_Stores_">Custom user stores</a></li>
+<li><a href="../../using-wso2-identity-server/configuring-user-stores">Custom user stores</a></li>
```
Comment (sherenewso2) [inline en/docs/setup/installation-prerequisites.md]: change this link path to "learn"

File: en/docs/setup/installation-prerequisites.md [hunk 10]
```diff
@@ -41,28 +35,31 @@ development platforms before starting the installation.
...
 <ul>
-<li>For environments that WSO2 products are tested with, see <a href="https://docs.wso2.com/display/TestedPlatforms/Environments+Tested+with+WSO2+Products">Environments Tested with WSO2 Products</a> .</li>
-<li>If you have difficulty setting up any WSO2 product in a specific platform or database, <a href="https://wso2.com/contact/">contact WSO2</a> .</li>
+<li>WSO2 Identity Server is shipped with a default embedded Apache DS that is suitable for development purposes. WSO2 does not recommend using Apache DS in a production environment due to scalability issues that exist with Apache DS.</li>
+<li>WSO2 does not recommend using the H2 database as a user store in production environments. However, you can use the H2 database for development purposes if necessary.</li>
+</ul>
+</p>
+</div>
+<ul> 
+<li>For environments that WSO2 products are tested with, see <a href="../../admin-guide/environments-tested-with-wso2-products">Environments Tested with WSO2 Products</a> .</li>
```
Comment (sherenewso2) [inline en/docs/setup/installation-prerequisites.md]: change this link path to "administer". Please check **all** relevant places and fix according to the new folder structure

File: en/docs/setup/installing-as-a-windows-service.md [hunk 11]
```diff
@@ -52,16 +42,13 @@ configuration for running a WSO2 product as a Windows Service.
...
-    with [secure
-    vault](https://docs.wso2.com/display/Carbon444/Carbon+Secure+Vault+Implementation)
-    .
-
-    **Minimal wrapper.conf configuration**
+    !!! info
+        If you wish to set additional properties from an external registry
+        at runtime, store sensitive information like usernames and passwords
+        for connecting to the registry in a properties file, and secure it
+        with [secure vault](../../admin-guide/carbon-secure-vault-implementation).
```
Comment (sherenewso2) [inline en/docs/setup/installing-as-a-windows-service.md]: change link path

File: en/docs/setup/installing-on-windows.md [hunk 12]
```diff
@@ -1,24 +1,17 @@
...
-    matrix](https://docs.wso2.com/display/compatibility/Tested+Operating+Systems)
-    to find out if this version of the product is fully tested on Windows.
-    
-    WSO2 Identity Server (WSO2 IS) 5.3.0 and 5.4.0 is
-    not supported on Windows 10, due to an issue with Windows 10 and Java.
-    We will update the information about Windows 10 support once the
-    incompatibility is resolved.
-    
+!!! tip "Before you begin"
+    See the [compatibility matrix](../../admin-guide/tested-operating-systems)
```
Comment (sherenewso2) [inline en/docs/setup/installing-on-windows.md]: change link path

File: en/docs/setup/installing-on-linux-or-os-x.md [hunk 13]
```diff
@@ -1,9 +1,7 @@
 # Installing on Linux or OS X
 
-!!! tip
-    
-    **Before you begin**, [please see our compatibility
-    matrix](https://docs.wso2.com/display/compatibility/Tested+Operating+Systems)
+!!! tip "Before you begin"
+    See the [compatibility matrix](../../admin-guide/tested-operating-systems)
```
Comment (sherenewso2) [inline en/docs/setup/installing-on-linux-or-os-x.md]: this link path should change

File: en/docs/setup/installing-on-solaris.md [hunk 14]
```diff
@@ -82,7 +80,6 @@ you can take the following approaches:
     sure to store sensitive data such as username and password to
     connect to the registry in a properties file instead of in the Java
     code and secure the properties file with the [secure
-    vault](https://docs.wso2.com/display/Carbon420/Carbon+Secure+Vault+Implementation)
-    .
+    vault](../../admin-guide/carbon-secure-vault-implementation).
```
Comment (sherenewso2) [inline en/docs/setup/installing-on-solaris.md]: this link path should change

File: en/docs/setup/installing-on-windows.md [hunk 15]
```diff
@@ -1,24 +1,17 @@
...
-    matrix](https://docs.wso2.com/display/compatibility/Tested+Operating+Systems)
-    to find out if this version of the product is fully tested on Windows.
-    
-    WSO2 Identity Server (WSO2 IS) 5.3.0 and 5.4.0 is
-    not supported on Windows 10, due to an issue with Windows 10 and Java.
-    We will update the information about Windows 10 support once the
-    incompatibility is resolved.
-    
+!!! tip "Before you begin"
+    See the [compatibility matrix](../../administer/tested-operating-systems)
```
Comment (sherenewso2) [inline en/docs/setup/installing-on-windows.md]: there is no file called "tested-operating-systems" in the administer folder. 
please check

File: en/docs/setup/installing-on-solaris.md [hunk 16]
```diff
@@ -1,22 +1,18 @@
 # Installing on Solaris
 
-!!! tip
-    
-    **Before you begin**, [please see our compatibility
-    matrix](https://docs.wso2.com/display/compatibility/Tested+Operating+Systems)
+!!! tip "Before you begin"
+    See the [compatibility matrix](../../administer/tested-operating-systems)
```
Comment (sherenewso2) [inline en/docs/setup/installing-on-solaris.md]: there is no file called "tested-operating-systems" in the administer folder.
please check

File: en/docs/setup/installing-on-linux-or-os-x.md [hunk 17]
```diff
@@ -1,9 +1,7 @@
 # Installing on Linux or OS X
 
-!!! tip
-    
-    **Before you begin**, [please see our compatibility
-    matrix](https://docs.wso2.com/display/compatibility/Tested+Operating+Systems)
+!!! tip "Before you begin"
+    See the [compatibility matrix](../../administer/tested-operating-systems)
```
Comment (sherenewso2) [inline en/docs/setup/installing-on-linux-or-os-x.md]: there is no file called "tested-operating-systems" in the administer folder.
please check

File: en/docs/setup/installation-prerequisites.md [hunk 18]
```diff
@@ -41,28 +35,31 @@ development platforms before starting the installation.
...
-<li>For environments that WSO2 products are tested with, see <a href="https://docs.wso2.com/display/TestedPlatforms/Environments+Tested+with+WSO2+Products">Environments Tested with WSO2 Products</a> .</li>
-<li>If you have difficulty setting up any WSO2 product in a specific platform or database, <a href="https://wso2.com/contact/">contact WSO2</a> .</li>
+<li>WSO2 Identity Server is shipped with a default embedded Apache DS that is suitable for development purposes. WSO2 does not recommend using Apache DS in a production environment due to scalability issues that exist with Apache DS.</li>
+<li>WSO2 does not recommend using the H2 database as a user store in production environments. However, you can use the H2 database for development purposes if necessary.</li>
+</ul>
+</p>
+</div>
+<ul> 
+<li>For environments that WSO2 products are tested with, see <a href="../../administer/environments-tested-with-wso2-products">Environments Tested with WSO2 Products</a> .</li>
+<li>If you have difficulty setting up any WSO2 product in a specific platform or database, <a href="https://wso2.com/contact/">contact WSO2</a>.</li>
```
Comment (sherenewso2) [inline en/docs/setup/installation-prerequisites.md]: there is no file called "environments-tested-with-wso2-products" in the administer folder.
please check

Comment (YvonneW) [reply en/docs/setup/installation-prerequisites.md]: Fixed

**Pull request merged**